test/Microsoft.AspNet.WebSockets.Client.Test/KestrelWebSocketHelpers.cs

```diff
@@ -45,20 +44,14 @@ public static IDisposable CreateServer(Func<HttpContext, Task> app)
             var config = new Configuration();
             config.Add(new MemoryConfigurationSource());
             config.Set("server.urls", "http://localhost:54321");
-            var services = HostingServices.Create(CallContextServiceLocator.Locator?.ServiceProvider, config)
-                .BuildServiceProvider();
-            var applicationLifetime = services.GetRequiredService<IApplicationLifetime>();
 
             var context = new HostingContext()
             {
-                ApplicationLifetime = applicationLifetime,
                 Configuration = config,
                 ServerFactoryLocation = "Kestrel",
-                ApplicationStartup = startup,
+                StartupMethods = new StartupMethods(startup, configureServices: null)
```

Comment:
@davidfowl Should we move the configure/configureServices to the context directly?  I don't think we gain much putting them together inside StartupMethods on the context at least.  We can still keep the class around for the Startup helpers